### PR TITLE
docs: fix typographic issues

### PR DESCRIPTION
**Pull request by 102**

### CODE REVIEW

File: docs/content/en/_index.md [hunk 1]
```diff
@@ -4,23 +4,23 @@ date: 2017-03-02T12:00:00-05:00
...
     image_path: /images/icon-shortcodes.svg
-    tagline: Hugo's shortcodes are Markdown's hidden superpower.
+    tagline: Hugo’s shortcodes are Markdown’s hidden superpower.
     copy: We love the beautiful simplicity of markdown’s syntax, but there are times when we want more flexibility. Hugo shortcodes allow for both beauty and flexibility.
 
   - heading: Built-in Templates
     image_path: /images/icon-built-in-templates.svg
     tagline: Hugo has common patterns to get your work done quickly.
-    copy: Hugo ships with pre-made templates to make quick work of SEO, commenting, analytics and other functions. One line of code, and you're done.
+    copy: Hugo ships with pre-made templates to make quick work of SEO, commenting, analytics and other functions. One line of code, and you’re done.
```
Comment (102) [inline docs/content/en/_index.md:23]: I'm not sure whether these different types of apostrophes were used on purpose, but it drove me mad having two types of them next to each other, as in example:

image

Please let me know if it makes sense to review the apostrophes across the other docs and/or take a look at some kind of automatised check for this issue.

File: docs/content/en/getting-started/installing.md [hunk 2]
```diff
@@ -205,7 +205,7 @@ Replace `brew install hugo` with `brew install hugo --HEAD` if you want the abso
 `brew` should have updated your path to include Hugo. You can confirm by opening a new terminal window and running a few commands:
 
 ```
-$ # show the location of the hugo executable
+# show the location of the hugo executable
```
Comment (102) [inline docs/content/en/getting-started/installing.md:208]: `$` is often used to emulate the shell console, but there are no other examples of such usage in the documentation, so it seems more consistent to remove this one

File: docs/content/en/getting-started/installing.md [hunk 3]
```diff
@@ -387,7 +387,7 @@ You'll need a place to store the Hugo executable, your [content][], and the gene
 3. Open Windows Command Line (cmd, "DOS") to add the `hugo.exe` executable to your PATH
     * do `set PATH=%PATH%;C:\Hugo\bin` to have hugo in PATH for the currently opened cmd box
     * do `setx PATH "%PATH%;C:\Hugo\bin"` to have hugo in PATH for every newly opened cmd box
-      * note: "setx", not "set", plus syntax 'key "val"', not 'key=val'
+      * note: `setx`, not `set`, plus syntax `key "val"`, not `key=val`
```
Comment (102) [inline docs/content/en/getting-started/installing.md:390]: CLI utils and args are usually written as code blocks, hence the change

**Review by coliff — APPROVED** [review]

**Comment by bep** [comment]
Thanks for this, but these commits needs to be in https://github.com/gohugoio/hugoDocs

I understand that that may be a hazzle for you, but I cannot merge this in here as that will create lot of manual merge work for me when I'm doing a new release. We store the docs in this repo as a Git sub tree and we're only making docs changes that's closely related to new features.

**Pull request closed without merging**